feature-libs/product/multi-d/occ/config/default-occ-variants-multidimensional-config.ts

```diff
@@ -4,8 +4,7 @@ export const defaultOccProductVariantsMultidimensionalConfig: OccConfig = {
   backend: {
     occ: {
       endpoints: {
-        product:
-          'products/${productCode}?fields=name,averageRating,stock(DEFAULT),description,summary,availableForPickup,code,url,price(DEFAULT),numberOfReviews,manufacturer,categories(FULL),priceRange,multidimensional,tags,images(FULL),purchasable,baseOptions(DEFAULT),baseProduct,variantOptions(DEFAULT),variantMatrix(DEFAULT)',
+        product: 'products/${productCode}?fields=FULL',
```

Comment:
Is this intended?

Reply:
yes we need this to full until API will be stable